examples/MultitenantBySubsections.xml

```diff
@@ -0,0 +1,172 @@
+                    <FloorAreaValue>5000</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+              <Subsection ID="Suite150">
+                <PremisesName>Suite 150</PremisesName>
+                <TenantID IDref="Tenant3"/>
+                <FootprintShape>Rectangular</FootprintShape>
+                <Story>1</Story>
+                <FloorAreas>
+                  <FloorArea>
+                    <FloorAreaType>Rentable</FloorAreaType>
+                    <FloorAreaValue>5000</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+              <Subsection ID="Suite155">
+                <PremisesName>Suite 155</PremisesName>
+                <FootprintShape>Rectangular</FootprintShape>
+                <Story>1</Story>
+                <FloorAreas>
+                  <FloorArea>
+                    <FloorAreaType>Rentable</FloorAreaType>
+                    <FloorAreaValue>2526</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+              <Subsection ID="Suite110">
+                <PremisesName>Suite 110</PremisesName>
+                <TenantID IDref="Tenant4"/>
+                <FootprintShape>Rectangular</FootprintShape>
+                <Story>1</Story>
+                <FloorAreas>
+                  <FloorArea>
+                    <FloorAreaType>Rentable</FloorAreaType>
+                    <FloorAreaValue>5000</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+              <Subsection ID="Suite200">
+                <PremisesName>Suite 200</PremisesName>
+                <FootprintShape>Rectangular</FootprintShape>
+                <Story>2</Story>
+                <FloorAreas>
+                  <FloorArea>
+                    <FloorAreaType>Rentable</FloorAreaType>
+                    <FloorAreaValue>25757</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+              <Subsection ID="Suite300">
+                <PremisesName>Suite 300</PremisesName>
+                <TenantID IDref="Tenant5"/>
+                <FootprintShape>Rectangular</FootprintShape>
+                <Story>3</Story>
+                <FloorAreas>
+                  <FloorArea>
+                    <FloorAreaType>Rentable</FloorAreaType>
+                    <FloorAreaValue>25000</FloorAreaValue>
+                  </FloorArea>
+                </FloorAreas>
+              </Subsection>
+            </Subsections>
+          </Facility>
+        </Facilities>
+      </Site>
+    </Sites>
+    <Contacts>
+      <Contact ID="Contact1">
+        <ContactRole>Occupant</ContactRole>
+        <ContactName>Arthur Dent</ContactName>
+        <ContactCompany>Colorado Phone Sanitizers Association</ContactCompany>
+        <ContactTelephoneNumber>
```

Comment:
I do think that we need a ContactTelephoneNumbers surrounding this element. So disregard my previous comment.